packages/datadog-instrumentations/src/kafkajs.js

```diff
@@ -77,10 +77,10 @@ addHook({ name: 'kafkajs', versions: ['>=1.4'] }, (obj) => {
           const innerAsyncResource = new AsyncResource('bound-anonymous-fn')
           return innerAsyncResource.runInAsyncScope(() => {
             const { topic, partition, message } = eachMessageArgs[0]
-            consumerStartCh.publish({ topic, partition, message })
+            const groupId = arguments[0].groupId
```

Comment:
Do you know if `groupId` is the same for all messages? Or can a publish operation with multiple messages have different `groupId`s? If the latter then we might need to extract the group ID from `eachMessageArgs` instead:

```js
            const { topic, partition, message, groupId } = eachMessageArgs[0]
```